.github/workflows/security-analysis.yml

```diff
@@ -1,16 +0,0 @@
-name: "Security Analysis"
```

Comment:
Why are we deleting the security analysis work flow?

Reply:
Sounds like a separate question from this feature. I think we should leave it in for now to not confuse the intent of this PR.

Reply:
It looks like [a run of that workflow](https://github.com/aurora-is-near/sputnikvm/actions/runs/13199174163/job/36847085014) was successful on a prior commit of this PR.